packages/stex-react-renderer/src/lib/PerSectionQuiz.tsx

```diff
@@ -15,6 +17,17 @@ import { ListStepper } from './QuizDisplay';
 import { getLocaleObject } from './lang/utils';
 import { ServerLinksContext, mmtHTMLToReact } from './stex-react-renderer';
 
+function extractProjectIdAndFilepath(problemId: string) {
+  const url = problemId.replace('http://mathhub.info/', '');
+  const parts = url.split('/');
+  const projectId = parts[0] + '/' + parts[1];
+  const filePath = parts
+    .slice(2, 5)
```

Comment:
These are not guaranteed to be 5. Then can be any number of parts. If less than or equal to 2, it is an error. Otherwise, the first two parts go into the projected and the remaining go into filepath